src/test/java/com/amazon/opendistroforelasticsearch/security/SlowIntegrationTests.java

```diff
@@ -159,4 +165,24 @@ public void testNodeClientDisallowedWithNonServerCertificate2() throws Exception
         }
     }
 
+    @Test
+    public void testDelayInSecurityIndexInitialization() throws Exception {
+        final Settings settings = Settings.builder()
+                .put(ConfigConstants.OPENDISTRO_SECURITY_ALLOW_DEFAULT_INIT_SECURITYINDEX, true)
+                .put("cluster.routing.allocation.exclude._ip", "127.0.0.1")
+                .build();
+        try {
+            setup(Settings.EMPTY, null, settings, false);
+        } catch (IOException e) {
+            // Index request has a default timeout of 1 minute, adding buffer between nodes initialization and cluster health check
+            Thread.sleep(1000*80);
+            clusterHelper.nodeClient().admin().cluster().updateSettings(
+                    new ClusterUpdateSettingsRequest().transientSettings(Settings.builder().put("cluster.routing.allocation.exclude._ip", "1.2.3.4").build()));
```

Comment:
set `"cluster.routing.allocation.exclude._ip"` to `null`.

Reply:
Putting `null` doesn't seem to be updating the filter setting. There seems to be some issue with deleting settings via persistent/transient.

Reply:
That does not sound right. It should be possible to reset settings to `null`. Please root cause and either set it to `null` or add comment why it can't be set to `null` and is set to `"1.2.3.4"`.

Reply:
Debugged further. A default setting cannot be removed. 

Reference: https://stackoverflow.com/questions/67534456/removing-an-elasticsearch-cluster-default-setting-dynamically/67534680#67534680
Also, tried simulating it myself. Applied a default setting via yml file. Tried removing it using transient/persistent setting. Does not get removed.

Reply:
I see, OK. Please add comment explaining why it can't be set to `null`. Also, if it needs to be set to some value, consider using invalid ip address or reserved ip address.

Reply:
IP validator prevents setting invalid IP address. Instead, will set a reserved IP address